.github/workflows/pre-release-tests.yml

```diff
@@ -0,0 +1,55 @@
+# Testing the code base against the MeiliSearch pre-releases
+name: Pre-Release Tests
+
+# Will only run for PRs and pushes to bump-meilisearch-v*
+on:
+  push:
+    branches: [bump-meilisearch-v*]
+  pull_request:
+    branches: [bump-meilisearch-v*]
+
+jobs:
+  integeration_tests:
+    name: integration-tests
+    runs-on: macos-11
+
+    env:
+      DEVELOPER_DIR: /Applications/Xcode_12.4.app/Contents/Developer
+      GITHUB_PAT: ${{ secrets.MEILIBOT_PAT_REPO }}
```

Comment:
Not used, we will add it once we use it.

Reply:
```suggestion
```

Reply:
I removed it in both tests and pre-release-tests, is this what you want?

Reply:
No, only here, no in `tests.yml`